nixos/modules/services/web-apps/grist-core.nix

```diff
@@ -0,0 +1,167 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib)
+    literalExpression
+    types
+    mkOption
+    mkDefault
+    mkPackageOption
+    mkEnableOption
+    mkIf
+    ;
+
+  cfg = config.services.grist-core;
+
+
+in
+{
+  meta.maintainers = with lib.maintainers; [ soyouzpanda ];
+
+  options.services.grist-core = {
+    package = mkPackageOption pkgs "grist-core" { };
+    enable = mkEnableOption "Grist core";
+
+    user = mkOption {
+      type = types.str;
+      default = "grist-core";
+      description = "User account under which grist-core runs.";
+    };
+
+    group = mkOption {
+      type = types.str;
+      default = "grist-core";
+      description = "Group under which grist-core runs.";
+    };
+
+    pythonEnv = mkOption {
```

Comment:
You should use a passthru in the package to do this, in grist-core you can do : 
```
passthru.pythonEnv = python3.withPackages (_: propagatedBuilInputs);
```
or similar, so that you don't write this list at two places, which can break if someone updates only one of them during a package upgrade

Reply:
Plus, as `visible = false` the example will not show up anywhere, so maybe remove it ?